rosdep/base.yaml

```
@@ -7791,6 +7791,19 @@ redis-server:
   gentoo: [dev-db/redis]
   nixos: [redis]
   ubuntu: [redis-server]
+robin-map:
```

Our typical process here is for these keys to match the name of the Ubuntu package, so this should be:

```suggestion
robin-map-dev:
```

---

All right. I could find many examples that contradict this, but it's not a problem at all for me so I will change it.

PS: 141 packages don't follow this convention:

ace:
acl:
antlr:
arista:
armadillo:
asio:
atlas:
binutils:
bullet-extras:
cdk:
cgal:
coinor-libcbc3:
collectd:
collectd-utils:
comedi:
cpuburn:
crypto++:
eigen2:
enblend:
enet:
f2c:
festival:
fmt:
freeimage:
geos:
glc:
glpk:
glut:
gnuplot:
gnuplot-iostream:
gperftools:
hdf5:
iwyu:
jack:
leveldb:
libapache2-mod-python:
libfftw3:
libgfortran3:
libgmp:
libgps:
libgstreamer-plugins-base0.10-0:
libgstreamer0.10-0:
libgstrtspserver-1.0-0:
libgtkmm:
libgts:
libignition-utils1:
libimage-exiftool-perl:
libjackson-json-java:
libjpeg:
libjson-glib:
libjson-java:
libjsonrpccpp-client0:
libjsonrpccpp-common0:
libjsonrpccpp-server0:
libjsonrpccpp-stub0:
libjsonrpccpp-tools:
libmagick:
libmicrohttpd:
libmockito-java:
libmongoc-1.0-0:
libnlopt0:
libnotify:
libogg:
libogre1.12.10:
libopenscenegraph:
liborocos-bfl:
libpcap:
libphonon:
libpocofoundation9:
libprocps6:
libqhull:
libqrencode:
libqt4-sql-psql:
libqwt6:
libreadline:
libstdc++6:
libsvm3:
libtesseract:
libunittest++:
libx11:
libxaw:
libxext:
libxml++-2.6:
libxml2:
libxmlrpc-c++:
libxrandr:
libxslt:
libxxf86vm:
linux-kernel-headers:
log4cplus:
lz4:
mapnik-utils:
matio:
mosquitto-clients:
mpfr:
mplayer:
mrpt:
msgpack:
multistrap:
muparser:
netcdf:
netpbm:
ninja-build:
nvidia-cuda:
nvidia-cudnn:
omniorb:
opende:
osm2pgsql:
osmium:
pcre:
pocketsphinx-bin:
poppler-utils:
portaudio:
proj:
r-base:
robin-map:
rti-connext-dds-6.0.1:
rtmidi:
sdformat13:
sdl:
sdl-gfx:
sdl-image:
sdl-mixer:
sdl-ttf:
sdl2:
sdl2-image:
sdl2-mixer:
sdl2-ttf:
setserial:
snappy:
sparsehash:
spdlog:
suitesparse:
tbb:
tinyxml2:
xdotool:
xerces:
xulrunner-1.9.2:
yaml:
zbar:
zziplib:

---

> All right. I could find many examples that contradict this, but it's not a problem at all for me so I will change it.

Yes, we introduced this rule a few years ago, but there are many old keys that don't follow it.  We can't really change them without breaking packages, so we tend to leave them in place.  And sometimes, we make a different choice for practical purposes.

In any case, for new keys we follow this rule where possible.